routes/api/products.js

```diff
@@ -0,0 +1,10 @@
+// const router = require("express").Router();
+const productsController = require('../../controllers/productsController');
```

Comment:
how come some one used const here but not above for the other file? Someone didnt code review well and i can tell. Not a good sign.

Reply:
removed the router, it is not use in controller at the moment.